README.md

```diff
@@ -41,3 +41,7 @@ These include file processing tools (in the file_processing directory). This cod
 - [ ] Write tests to ensure the functionality of the library
 - [ ] Add support for more file types (.docx, .xlsx, .pptx, etc.)
 - [ ] Add more metrics for comparison between files (ex. jaccard)
+
+## Build exe
+- Install `pyinstaller`
```

Comment:
should we just add this to the venv requirements file?

Reply:
great point, just made the change